contrib/kube-prometheus/assets/prometheus/rules/kubernetes.rules.yaml

```diff
@@ -84,3 +84,24 @@ groups:
     annotations:
       description: No API servers are reachable or all have disappeared from service
         discovery
+
+  - alert: K8sCertificateExpirationNotice
+    labels:
+      severity: warning
+    annotations:
+      description: Kubernetes API Certificate is expiring soon (less than 7 days)
+    expr: sum(apiserver_client_certificate_expiration_seconds_bucket{le="604800"}) > 0
+
+  - alert: K8sCertificateExpirationLastDayNotice
```

Comment:
Typically alert names are chosen the same, with different severity, that way we can inhibit the less severe alerts when an equally named one is firing, so this would also be `K8sCertificateExpirationNotice`.